src/app/providers/LocalSettingsContext.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react'
+import { LocalSettings } from '../../types/local-settings'
+
+export interface LocalSettingsProviderContext {
+  readonly settings: LocalSettings
+  changeSetting: (key: keyof LocalSettings, value: any) => void
```

Comment:
No typesafety :scream:

Reply:
Thats a copy-paste error, what we wanted (and really have in the implementation) is this: `<T extends keyof LocalSettings>(key: T, value: LocalSettings[T]) `

Will fix in the next related PR.

Reply:
Fixed in https://github.com/oasisprotocol/explorer/pull/1794/commits/0748a432f74967d5810695c9e84cc5dca0c92cd3